### PR TITLE
SW-4713 Add participant project option menu that includes export, for now a no-op until BE is done

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2388** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2388?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2387** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2387?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/AcceleratorRouter/ParticipantProjects/SingleView.tsx [hunk 1]
```diff
@@ -1,35 +1,57 @@
...
       <Box display='flex' flexDirection='row' flexGrow={0} marginRight={theme.spacing(3)} justifyContent='right'>
-        <Button
-          id='editProject'
-          icon='iconEdit'
-          label={strings.EDIT_PROJECT}
-          priority='primary'
-          onClick={() => goToParticipantProjectEdit(projectId)}
-          size='medium'
-          type='productive'
+        {isAllowed('UPDATE_PARTICIPANT_PROJECT') && (
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/ParticipantProjects/SingleView.tsx:30]: move this above the `<Box>` ? maybe return undefined if isAllowed is false.

**Review by karthikbtf — APPROVED** [review]
Nice

**Comment by nickgraz** [comment]
### Merge activity

* **Mar 21, 7:53 PM EDT**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2387).

**Pull request merged**